### PR TITLE
my trials project

### PR DESCRIPTION
**Pull request by Nicolaswg**
this is my second project of the trials

### CODE REVIEW

**Review by Nicolaswg — COMMENTED** [review]
# Congratulation 🥇 

You are working hard but you are required to make some changes to finally complete de project.

## Required Changes ♻️ 

###Profecional Repository
For good practices, you should make your repository looks professional and organizes.
 
1. Make your README.md more presentable with a specific description of the entire project.
2. Put screenshots of the last view of your webpage side
3. Make a more descriptive pull request and git commit messages to all the changes you made one by one fill.

Follow the comment below instructions to make a professional and successful project.

*Optional changes are not required but you should hear them*

## Linters Errors 🔴 

Submitting  projects with existing errors are not professional practice, you should first:

- See if all Linters fills are successfully complete.
- read and understand your code changes.
- Open an issue if there is any bug that you couldn't solve.

**Review by D4nielJ — CHANGES_REQUESTED** [review]
## Hello @Nicolaswg, 

You're doing great! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights :fire: 
- Your comments are well redacted
- You always explain the "why" behind it.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

 - Using gifs and emojis makes our reviews more human-friendly. :alien: :robot: 

## Happy coding and stay awesome! 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    <title>FCC: Survey</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <div class="item-container">
+      <header class="header">
+    <h1 id="title">freeCodeCamp Survey Form</h1>
+    <p id="description">Thank you for taking the time to help us improve the platform</p>
+  </header>
```
Comment (Nicolaswg) [inline index.html:14]: I think that for a better HTML code structure, you could put your header tag outside the div container 🏷️

File: index.html [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+
```
Comment (Nicolaswg) [inline index.html:16]: Empty newlines make your code longer without a reason. I suggest you to only use newlines when is required.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+          <textarea name="" id="text-area" placeholder="Enter your comment here..."></textarea>
+        </div>
+        <div class="button-container">
+          <input type="submit" id="submit" class="button-style">Submit</input>
+        </div>
+      
+      </form>
+    </div>
+    
+    <script  src="js/index.js"></script>
```
Comment (Nicolaswg) [inline index.html:106]: It is important to know that it is not necessary to link files that are not in your workflows.  I suggest you to only link the files you are working on. 🎉

File: index.html [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+        <input type="radio" value='Definitely'  id="radio1" name='user-recommend'>
+
+        <label for="radio2">Maybe</label>
+        <input type="radio" value='Maybe'   id="radio2" name='user-recommend'>
+
+        <label for="radio3">Not sure</label>
+        <input type="radio" value='Not sure'   id="radio3" name='user-recommend'>
+      </div>
+
+
```
Comment (Nicolaswg) [inline index.html:57]: Empty newlines make your code longer without a reason. I suggest you to only use newlines when is required.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+        <label for="check11">
+        <input type="checkbox" value="Additional Courses" id="check11">Additional Courses     </label>
+      </div>
+        
+        <div class="data-container4">
+          <p>Any comments or suggestions?</p>
+          <textarea name="" id="text-area" placeholder="Enter your comment here..."></textarea>
+        </div>
+        <div class="button-container">
+          <input type="submit" id="submit" class="button-style">Submit</input>
```
Comment (Nicolaswg) [inline index.html:100]: for buttons, I think it is better to use an 'a' or 'button' tag. 👍

**Review by Nicolaswg — COMMENTED** [review]
# New re-work Comments 🥇

**Review by alonsofalconi — APPROVED** [review]
Hi @Nicolaswg,

Your project is complete!  You have performed a great review :shipit: 
Congratulations! 🎉 

## To highlight :dart:
- Great use of inline comments :heavy_check_mark:
- Use of emojis to make the communication more human :heavy_check_mark:
- Suggests instead of giving orders :heavy_check_mark:

## To improve :memo:
- You have missed checking if the linters are passing :warning:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Nicolaswg** [comment]
@alonsofl  the linters error should I put it as a comment or find the error ?

**Comment by alonsofalconi** [comment]
@Nicolaswg, when a PR has linters errors it is a reason to not perform the code review because it is the first check. It is like the PR is already telling the code owner that there is a bug :wink:

**Review by VanessaAoki — COMMENTED** [review]
Hi @Nicolaswg, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a review in a Pull Request that was already merged. If you are uncertain on how to request for a PR, you can read more about it [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/how_to_ask_for_a_code_review.md). If you are requesting a new review, it should be done in it's own Pull Request.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding! 👩‍💻 

keep trying

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**